src/components/adjudications/AdjudicationForm.jsx

```diff
@@ -50,19 +50,14 @@ class AdjudicationForm extends React.Component {
     const { 
       adjudicationId, 
       collectionName, 
-      currentValues : {
-        audioURL, 
-        judgeName, 
-        notes
-      } 
+      currentValues
     } = this.props;
     const { artisticMark, technicalMark } = this.state;
     const cumulativeMark = (parseInt(artisticMark, 10) + parseInt(technicalMark, 10)) / 2;
+
     const data = {
-      audioURL,
       cumulativeMark,
```

Comment:
If i recall correctly, I had a bug running taking this spread object. 

The value of cumulativeMark in this object comes from the value of this.state, then currentValues, then this cumulativeMark value. The way the spread operator works is you're defining an object, saying add and overwrite everything so far with currentValues, then add/overwrite with this.state. 

We need to move cumulativeMark to the bottom of this object declaration.